src/helpers/GetDonationDate.ts

```diff
@@ -3,7 +3,20 @@ import { ethers } from "ethers";
 import { addresses } from "../constants";
 import { IBaseAddressRecipientAsyncThunk } from "../slices/interfaces";
 
+/**
+ * Gets the date of the first time a user donated to a specific recipient
+ * @param address Current user's address
+ * @param recipient The donation target of the current user
+ * @param networkID ID number of the network the user is currently connected to
+ * @param provider Network provider object
+ * @returns String representation of the date as Month Day, Year
```

Comment:
Why return this as a string? Why not a native Date?

If you want to return it as a string, document the format that is used.

Reply:
It felt like the simplest route to convert it to a good display format if the same display format was acceptable everywhere we would be using it. But, I'm not married to doing it that way. Do you think it's better to return as a Date and convert at the point of use?

Reply:
For now just documenting format